dashboard/config/locales/scripts.en.yml

```diff
@@ -1830,3 +1830,10 @@ en:
           Why Computer Science belongs in Algebra: Why Computer Science belongs in Algebra
           Evaluation Blocks: Evaluation Blocks
           Teacher Dashboard: Teacher Dashboard
+        plc-cspunit3:
```

Comment:
Yep, these are the i18n strings for `plc-cspunit1.script`.  Even though the script isn't localized we still need the English strings.